### DISCUSSION TITLE
my hub is detected as an storage device

### DISCUSSION THREAD

**Discussion by iamnowcodingforfun** [pybricks/support#1336] [Q&A]
so how can i fix this?
can i change it or not?

**Comment by dlech** [comment]
Can you please give more details about the problem? Which hub are you using? Which firmware is installed on the hub? What is the exact message you are seeing?

**Reply by iamnowcodingforfun** [reply]
the default lego hub software, i cant seem to even find it in the device manager while it is connected to my pc, i also dont see the universel serial bus devices page

**Reply by dlech** [reply]
If you are using the official LEGO firmware with SPIKE Prime or MINDSTORMS Robot Inventor, then it should show up under COM ports in the device manager.

**Reply by iamnowcodingforfun** [reply]
well i just checked and there is nothing saying lego, can it maby stand under another name?
